deploy/helm/Chart.yaml

```diff
@@ -6,7 +6,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.5.0
+version: 0.6.0
```

Comment:
no need to change the chart version , we are automatically bumping it before we do a release